SD_Assignment1/src/test/java/ro/utcn/aronTest/SdAssignment1ApplicationTests.java

```diff
@@ -0,0 +1,98 @@
+package ro.utcn.aronTest;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.stream.Collectors;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import ro.utcn.aron.persistence.api.RepositoryFactory;
+import ro.utcn.aron.persistence.memory.InMemoryRepositoryFactory;
+import ro.utcn.aron.persistence.memory.InMemoryUserRepository;
+import ro.utcn.aron.service.QuestionManagementService;
+import ro.utcn.aron.service.UserManagementService;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ContextConfiguration(classes = { InMemoryUserRepository.class })
+public class SdAssignment1ApplicationTests {
+
+	RepositoryFactory memRepositoryFactory = new InMemoryRepositoryFactory();
+
+	@Test
+	public void testAddQuestion() {
+		QuestionManagementService questionManagementService = new QuestionManagementService(memRepositoryFactory);
+		questionManagementService.addQuestion("Save Question Test", "I'm testing save question method", "test,memory",
+				"Paul");
+
+		assertEquals( questionManagementService.listQuestions().size() , 1);
+	}
+
+	@Test
+	public void testUserLogin() {
+		UserManagementService userManagementService = new UserManagementService(memRepositoryFactory);
+		userManagementService.save("Rose", "abc123");
+
+		assertEquals( userManagementService.matches("Rose", "abc123"), true);
```

Comment:
There is an `assertTrue` method that you can use instead of `assertEquals(true, ...)`.